packages/next/package.json

```diff
@@ -104,8 +104,8 @@
   "peerDependencies": {
     "@opentelemetry/api": "^1.1.0",
     "@playwright/test": "^1.41.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "19.0.0-beta-4508873393-20240430",
```

Comment:
We should use "^19.0.0-beta.0"?

Reply:
Those versions don't exist.

Reply:
It means >= 19.0.0-beta.0, so it can match all the versions above for pages router

Reply:
Here are the 19.0 versions in ASCII sort order

```
  "19.0.0-beta-1beb73de0f-20240503",
  "19.0.0-beta-4508873393-20240430",
  "19.0.0-beta-5d29478716-20240506",
  "19.0.0-beta-73bcdfbae5-20240502",
  "19.0.0-beta-94eed63c49-20240425",
  "19.0.0-beta-b498834eab-20240506",
```
You can see that it's not necessarily in chronological order.

[semver.org](https://semver.org/) section 11.4.2 says ASCII sort order will be used for everything after the last dot.

Reply:
Actually I think `19.0.0-canary-*` would match as latest in that range. Because `c > b`. I think there needs to be dot-separated tags like `19.0.0.beta.484747` in order to only match the `beta`s.